docs/CHANGELOG.md

```diff
@@ -48,10 +48,18 @@ Help: "The total number of eth node connection errors",
 
 - Support for arbitrum
 
+- Experimental: Add `DATABASE_BACKUP_MODE`, `DATABASE_BACKUP_FREQUENCY` and `DATABASE_BACKUP_URL` configuration variables
+
+It's now possible to configure database backups: on node start and separately, to be run at given frequency.
+
+DATABASE_BACKUP_MODE enables the initial backup on node start (either full or partial). Additionally, if DATABASE_BACKUP_FREQUENCY variable
```

Comment:
Better actually list the modes. `none`, `lite` or `full`

Reply:
It might be nice to see a couple of examples here with explanations so nops can play with it on their own